src/fosslight_util/parsing_yaml.py

```diff
@@ -7,11 +7,14 @@
 import logging
 import codecs
 import os
+import re
 import sys
 from .constant import LOGGER_NAME
 from .oss_item import OssItem
 
 _logger = logging.getLogger(LOGGER_NAME)
+SUPPORT_OSS_INFO_FILES = ["oss-pkg-info.yaml", "sbom-info.yaml"]
+EXAMPLE_OSS_PKG_INFO_LINK = "https://github.com/fosslight/fosslight_prechecker/blob/main/tests/convert/oss-pkg-info.yaml"
 
```

Comment:
It is suggested to rename the file fosslight_prechecker/blob/main/tests/convert/oss-pkg-info.yaml to sbom-info.yaml and also change this link.